modules/mbedtls/configs/config-mbedtls.h

```diff
@@ -41,8 +41,8 @@
 #define MBEDTLS_HAVE_ASM
 #endif
 
-#if defined(CONFIG_MBEDTLS_LMS)
-#define MBEDTLS_LMS_C
+#if defined($MBEDTLS_LMS)
+#define $MBEDTLS_LMS
```

Comment:
I'm not really following what this even does, if the variable is defined, it defines the same variable... And it's got a `$` as the first character which looks like a mistake? And it doesn't have a `CONFIG_` prefix in the `if` which looks like a mistake?